app/components/ui/NewsArticles/NewsArticles.module.scss

```diff
@@ -0,0 +1,277 @@
+@import "app/styles/utils/_helpers.scss";
+
+$carouselButtonDimension: 40px;
+$carouselButtonOffset: $carouselButtonDimension * 2 + 5px;
+
+.container {
+  background-color: #f5f5f5;
+  padding: 15px 0 25px;
+  &.carouselCollapsed {
+    padding-bottom: 1px;
+  }
+}
+
+.articlesHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 90%;
+  max-width: 1160px;
+  margin: auto;
+  @media screen and (max-width: $break-tablet-p) {
+    width: calc(100% - $carouselButtonOffset);
+    max-width: 1160px;
+  }
+}
+
+.title {
+  font-weight: 700;
+  font-size: 26px;
+  @media screen and (max-width: $break-tablet-p) {
+    font-size: 24px;
+  }
+}
+
+.toggleCarouselButton {
+  background-color: transparent;
+  height: $carouselButtonDimension;
+  width: $carouselButtonDimension;
+
+  // Button overrides
+  padding: 0;
+  &:focus,
+  &:hover {
+    background-color: none;
+    box-shadow: none;
+  }
+}
+
+.toggleCarouselButton {
+  .expandCarouselChev,
+  .collapseCarouselChev {
+    font-size: 28px;
+    color: rgba(0, 0, 0, 0.6);
+  }
+}
+
+.expandCarouselChev {
+  transform: rotate(180deg) scaleX(-1);
+}
+
+.carouselContainer {
+  max-width: 1160px;
+  margin: auto;
+  width: 90%;
+  margin-bottom: 9px;
+  opacity: 1;
+  transform: scale(1);
+  animation: grow 0.3s ease-in-out;
```

Comment:
This renders weirdly on my computer (macOS with Chrome 112). When I collapse the breaking news component, it collapses instantaneously, with no animation, but when I expand it, it expands with kind of a weird animation where the gray background instantly appears, but then the white boxes grow gradually with an animation.

I'm not too familiar with animation keyframes, but the approach I've used for collapsing animations in the past was the `max-height` + `transition: max-height ...` styles that the article I linked to mention.

Not sure if you're experiencing something different on your computer, but I just wanted to point it out.

Reply:
I went into a little animation hole and added some transitions using max-height and opacity. Now the entire Breaking News element animates and it looks smoother. I also added an animation for when the element initially appears on screen after the data has been fetched; I used a class rather than returning null if there are no news articles. See animation here: [BreakingNewsDraft](https://www.loom.com/share/9a89fd614bc94322b22812740bd4a3d5)
I will demo this at the product meeting tomorrow to see what folks think about the animations and can adjust as needed before going to prod.

Reply:
Awesome, that looks great to me!